plugins/vli/fu-vli-usbhub-device.c

```diff
@@ -644,6 +657,22 @@ fu_vli_usbhub_device_probe(FuDevice *device, GError **error)
 	} else {
 		fu_device_set_summary(device, "USB hub");
 	}
+
+	/* check for the DA300 which does not respond well to probing */
+	if ((fu_usb_device_get_vid(FU_USB_DEVICE(device)) == 0x2109 &&
+	     fu_usb_device_get_pid(FU_USB_DEVICE(device)) == 0x2820 &&
+	     _fu_usb_device_has_child(FU_USB_DEVICE(device), 0x208E, 0x0001)) ||
+	    (fu_usb_device_get_vid(FU_USB_DEVICE(device)) == 0x2109 &&
+	     fu_usb_device_get_pid(FU_USB_DEVICE(device)) == 0x0820 &&
+	     _fu_usb_device_has_child(FU_USB_DEVICE(device), 0x0BDA, 0x8153))) {
```

Comment:
Potentially, yes. The first is fine as it's a proper vendor VID. The 2nd is for the USB3 interface (the chip exports two devices to fwupd; one USB2 and one USB3) and I think the hub is going to fail if we probe on this one too. I'm not super happy about the 2nd one either, but I think the USB3 interface only has that ethernet device under it, and no parent.

Reply:
I guess @CragW can you confirm if it fails on probing both?  It would be ideal to only avoid probing the first if we can for the reason described above.

Reply:
We have to avoid probing the 2nd section i.e.: `2109:0820` too otherwise spi conversation still breaks the hub. How about extra requirement on port number `2`? Comparing with `g_usb_device_get_port_number` to determine the dependency along a specific location.

Reply:
I've just tested it which looks good here, if that helps to mitigate the impact to real world cases. 

```
+            _fu_usb_device_has_child(FU_USB_DEVICE(device), 0x208E, 0x0001, 0)) ||
            ...
            ...
+            _fu_usb_device_has_child(FU_USB_DEVICE(device), 0x0BDA, 0x8153, 2))) {
```

```
+               if (g_usb_device_get_vid(usb_device) == vid &&
+                   g_usb_device_get_pid(usb_device) == pid) {
+                       if ((portidx == 0) || (g_usb_device_get_port_number(usb_device) == portidx))
+                               return TRUE;
+               }
```

Reply:
I can't tell you how I know, but firmware REV 3003 is specific to this hardware. I think, if it works, https://github.com/fwupd/fwupd/pull/4324 is a better way to fix this issue as it wont cause regressions on other hardware using the same VID&PID.

Reply:
I'm not sure others but yes you're right DA300 is using REV 3003, let's go ahead with #4324 .